protostar/configuration_file/configuration_file.py

```diff
@@ -26,22 +26,28 @@ def set_section(
         data: dict[str, Any],
         profile_name: Optional[str] = None,
     ) -> None:
-        pass
+        ...
 
     @abstractmethod
     def build(self) -> str:
         ...
 
+    @abstractmethod
+    def get_file_extension(self) -> str:
```

Comment:
This class is a **Content**Builder` i don't thing this method belongs here, file extension is not a content

Reply:
`getContentFormat`?

Reply:
better

Reply:
Done in [3612d93](https://github.com/software-mansion/protostar/pull/897/commits/3612d9376d39418b26cb3aa1c124ad5cc83fbe97)